### ISSUE TITLE
about the Data

### ISSUE THREAD

**Issue by xiaoiker** [js3611/dAUTOMAP#2]
Hello,

Thanks very much for open source your work, can I ask I question?
Do you do some data preprocessing for the inputs? For example normalize them to 0~1?

Thanks a lot!

**Comment by Aditya-Tejaswi** [comment]
Hey,

Thanks for introducing the concept and ideas behind the dAUTOMAP :+1: . 

I'm trying to implement the dAUTOMAP using tensorflow. As @xiaoiker mentioned, I would also like to know about the pre-processing steps. Currently, I'm training the model for 64 x 64 bit images acquired from ImageNet and it is overfitting whereas the AUTOMAP model doesn't. AUTOMAP is giving me good reconstructions but the dAUTOMAP seems to be overfitting despite the BatchNormalization layers. 

The only preprocessing step I do is dividing the images by 127.5 and subtracting 1 from all of the pixels before the FFT. I would really appreciate any insights into this :smile:
